parm/config/gefs/config.fcst

```diff
@@ -242,7 +242,11 @@ export FSICS="0"
 
 #---------------------------------------------------------------------
 # Write more variables to output
-export DIAG_TABLE="${PARMgfs}/ufs/fv3/diag_table"
+if (( OFFSET_START_HOUR != 0 )); then
+  export DIAG_TABLE="${PARMgfs}/ufs/fv3/diag_table_replay"
+else
+  export DIAG_TABLE="${PARMgfs}/ufs/fv3/diag_table"
+fi
```

Comment:
can this be conditioned on `REPLAY_ICS`?

Reply:
Sure. Done.